### ISSUE TITLE
Record peoples' timezones

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#2278]
We should record peoples' time zones automatically because we're going to need to:

* send email notifications at a fixed local time (say, 5pm or 8am) when we start sending daily or weekly digests (see #1421 and others in this milestone)
* resolve #2242 (google calendar integration) by using the node author's default local timezone unless one is specified (currently events are posted to GCal 1 day early since they're at GMT time)

This has some guidance on this: http://nithinbekal.com/posts/rails-user-timezones/

I was hoping we could do this initially with a button on the profile page, that detects time zone and saves it as a profile tag like `timezone:-06:00` or `timezone:+4:30` which we could parse for later use. 

On the profile page, we should be able to use `addTag(tagname)`: 

https://github.com/publiclab/plots2/blob/24b67d65f2bd7ec915c83ccbde4566796fede0b3/app/views/tag/_replication.html.erb#L24

Later, we would want to add a follow-up issue of using the `timezone` tag in https://github.com/publiclab/plots2/blob/master/app/views/tag/rss.rss.builder to set the default timezone, using Ruby time zone methods: 

http://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html#method-i-iso8601

**Comment by amitsin6h** [comment]
want to work on this issue as no first-timer-only found. @jywarren

**Comment by Gauravano** [comment]
hi @amitsin6h you can take this issue or you can see [here](https://publiclab.github.io/community-toolbox/#r=all) for fto candidate(they are close to first-timer issue).

**Comment by amitsin6h** [comment]
@Gauravano also most of the fto issues are also taken

**Comment by Gauravano** [comment]
If you can solve this one that would be great !!

**Comment by f3mshep** [comment]
@amitsin6h Are you still looking into this? Would you be interested in pair programming if you are stuck?

**Comment by amitsin6h** [comment]
@f3mshep yeah sure. I have some issues also , I can also discuss with you.

**Comment by jywarren** [comment]
Hi, all -- let's try to solve this in a new controller method in the users_controller, something like `def timezone` -- where we could use code similar to this to create the user tag in the format above:

https://github.com/publiclab/plots2/blob/master/app/controllers/user_tags_controller.rb#L19-L26

What do you think?
